cuqi/implicitprior/_regularizedGaussian.py

```diff
@@ -206,16 +256,18 @@ def sqrtcov(self):
     def sqrtcov(self, value):
         self.gaussian.sqrtcov = value     
     
-    def get_conditioning_variables(self):
-        return self.gaussian.get_conditioning_variables()
```

Comment:
Why is this removed?

Reply:
The first thing that `get_conditioning_variables` does is ask for `get_mutable_variables`. So as long as `get_mutable_variables` and the attributes are correct, this wrapping should not be needed anymore.

Reply:
You are right!